### PR TITLE
[Improve][doc] Add doc name check

### PR DESCRIPTION
**Pull request by zhangshenghang**
### Purpose of this pull request

Verify whether the Chinese and English names of the markdown file are consistent

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownHeaderTest.java [hunk 1]
```diff
@@ -41,6 +41,43 @@ public static void setup() {
...
+                enFileName.stream()
+                        .map(String::toLowerCase)
+                        .filter(
+                                name ->
+                                        zhFileName.stream()
+                                                .map(String::toLowerCase)
+                                                .anyMatch(name::equals))
+                        .count();
+
+        Assertions.assertEquals(equalCount, equalIgnoreCaseCount);
```
Comment (Hisoka-X) [inline seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownHeaderTest.java]: Can we throw exception of the chinese doc name can not be found in english version when doc not match?

File: seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownHeaderTest.java [hunk 2]
```diff
@@ -41,6 +41,43 @@ public static void setup() {
         docsDirectorys.add(Paths.get("..", "docs", "zh"));
     }
 
+    @Test
+    public void testFileNameVerify() {
```
Comment (Hisoka-X) [inline seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownHeaderTest.java]:
```suggestion
    public void testChineseDocFileNameContainsInEnglishVersionDoc() {
```

File: docs/en/contribution/how-to-create-your-connector.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Develop Your Own Connector
```
Comment (Hisoka-X) [inline docs/en/contribution/how-to-create-your-connector.md:1]: Please update https://github.com/apache/seatunnel/blob/3a54f1253f957b6004d616560c52dedf3638c2c0/docs/sidebars.js#L223 too.

**Review by Hisoka-X — DISMISSED** [review]

**Comment by Hisoka-X** [comment]
Thanks @zhangshenghang !

**Review by hailin0 — DISMISSED** [review]

File: seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownTest.java [hunk 4]
```diff
@@ -41,6 +41,52 @@ public static void setup() {
         docsDirectorys.add(Paths.get("..", "docs", "zh"));
     }
 
+    @Test
+    public void testChineseDocFileNameContainsInEnglishVersionDoc() {
```
Comment (Hisoka-X) [inline seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownTest.java:48]: Please disabled the case on windows to fix ci.

File: docs/en/contribution/how-to-create-your-connector.md (hunk 3 above)
Comment (zhangshenghang) [reply docs/en/contribution/how-to-create-your-connector.md:1]: modify completed

File: seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownTest.java (hunk 4 above)
Comment (zhangshenghang) [reply seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownTest.java:48]: modify completed

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**